.github/workflows/scan_secrets-git.yml

```diff
@@ -1,17 +1,21 @@
-# Secret security scan
-name: Secret security scan
+name: Secrets security scan
 
 on:
   workflow_call:
+    secrets:
+      GITLEAKS_LICENSE:
+          description: 'Organization gitleaks license'
+          required: true
 
 jobs:
-  security-scan:
+  scan-secrets:
     runs-on: ubuntu-latest
-    name: Security scan
+    name: gitleaks
     steps:
       - uses: actions/checkout@v3
-      - name: scan-secrets
-        run: |
-          DOCKER_USER_ID="$(id -u)"
-          docker image pull zricethezav/gitleaks:v8.8.4
-          docker run -u ${DOCKER_USER_ID} -v ${GITHUB_WORKSPACE}:/path zricethezav/gitleaks:v8.8.4 detect --source="/path" -v
\ No newline at end of file
+        with:
+          fetch-depth: 0
+      - uses: gitleaks/gitleaks-action@v2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITLEAKS_LICENSE: ${{ secrets.GITLEAKS_LICENSE }}
```

Comment:
Please note that if you are scanning repos that belong to an organization, you'll have to [acquire a GITLEAKS_LICENSE](https://github.com/gitleaks/gitleaks-action#environment-variables) to use v2 (free "Trial" license available). That might come as a surprise to my users that are accustomed to using Gitleaks-Action free of charge, so I wrote a blog post explaining how/why I decided to monetize this project: https://blog.gitleaks.io/gitleaks-llc-announcement-d7d06a52e801

Reply:
Já coloquei a licença nos segredos da organização.